lib/matplotlib/contour.py

```diff
@@ -1674,7 +1676,8 @@ def _initialize_x_y(self, z):
           contour(X,Y,Z,V)
 
         draw contour lines at the values specified in sequence *V*,
-        which must be in increasing order.
+        which must be in increasing order for
+        :func:`~matplotlib.pyplot.contourf`.
```

Comment:
Here and below, I think the docs would be more readable without the sphinx link syntax.  Isn't it just linking back to what the reader is already reading?